.github/workflows/llvm-tests.yml

```diff
@@ -58,10 +59,17 @@ jobs:
       - name: Setup Variables
         id: vars
         run: |
-          if [ ${{ steps.version.outputs.LLVM_VERSION_MINOR }} -ne 0 ] || [ ${{ steps.version.outputs.LLVM_VERSION_PATCH }} -eq 0 ]; then
+          # libLLVM:
+          # 18.1.0 we aren't doing ABI checks.
+          # 18.1.1 We want to check 18.1.0.
+          # libclang:
```

Comment:
Isn't libclang handled by the separate libclang-abi-tests workflow?

I think you probably meant the LLVM C API rather than libclang here?